### PR TITLE
change project to run with pipenv

### PR DESCRIPTION
**Pull request by deluge**
I swapped the test-env to pipenv. Please have a look at it and if everthing is fine, could you setup the repo in codecov? I prepared it in the travis config file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-.tox
-coverage.xml
+*.egg-info
+.DS_Store
 .coverage
 .cache
-*.pyc
-*__pycache__*
+.pytest_cache
+didadata/settings.py
```
Comment (stephrdev) [inline .gitignore]: Is this line really required?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,7 @@ Changelog
 ------------------
 
 * Fix: import DjangoFilterBackend from django-filters `see django-rest-framework  <https://www.django-rest-framework.org/community/3.5-announcement/#djangofilterbackend>`_
+* Swapped test environment to pipenv
```
Comment (stephrdev) [inline CHANGELOG.rst]:
```suggestion
* Refactored project to use pipenv for testing and development
```

File: MANIFEST.in [hunk 3]
```diff
@@ -5,5 +5,7 @@ include setup.cfg
 
 recursive-include didadata *.py
 recursive-include didadata/locale *
-recursive-exclude didadata/tests *
+recursive-exclude tests *
+recursive-exclude examples/ *
```
Comment (stephrdev) [inline MANIFEST.in]: We don't have examples, therefore not required?

File: README.rst [hunk 4]
```diff
@@ -1,18 +1,36 @@
...
+
+Prepare for development
+-----------------------
+
+A Python 3.6 interpreter is required in addition to pipenv.
+
+.. code-block:: shell
+
+    $ pipenv install --python 3.6 --dev
+    $ pipenv install -e .
```
Comment (stephrdev) [inline README.rst]: editable "." is already part of the pipfile, no need to install this manually.

File: didadata/__init__.py [hunk 5]
```diff
@@ -0,0 +1,2 @@
+__version__ = '0.3.1'
+default_app_config = 'didadata.apps.DidadatalConfig'
```
Comment (stephrdev) [inline didadata/__init__.py]:
```suggestion
default_app_config = 'didadata.apps.DidadataConfig'
```

File: didadata/apps.py [hunk 6]
```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
+
+
+class DidadatalConfig(AppConfig):
```
Comment (stephrdev) [inline didadata/apps.py]:
```suggestion
class DidadataConfig(AppConfig):
```

File: setup.py [hunk 7]
```diff
@@ -1,87 +1,41 @@
...
+    long_description = f.read()
 
 
 setup(
     name='django-didadata',
+    version=VERSION,
     description='A Django app to collecto numeric data.',
-    long_description=read('README.rst'),
-    version=version,
-    license='BSD',
```
Comment (stephrdev) [inline setup.py]: Why was the line removed?

File: setup.py [hunk 8]
```diff
@@ -1,87 +1,41 @@
...
-    'sphinx_rtd_theme'
-]
+with open(os.path.join(BASE_DIR, 'README.rst'), encoding='utf-8') as f:
+    long_description = f.read()
 
 
 setup(
     name='django-didadata',
+    version=VERSION,
     description='A Django app to collecto numeric data.',
```
Comment (stephrdev) [inline setup.py]:
```suggestion
    description='A Django app to collect numeric data.',
```

File: setup.py (hunk 7 above)
Comment (deluge) [reply setup.py]: I have put long_description and version on the top and forgot the license... will be added.

File: .gitignore (hunk 1 above)
Comment (deluge) [reply .gitignore]: This line should be removed

**Comment by deluge** [comment]
Hey @stephrdev please have a look once again :)

**Comment by deluge** [comment]
Aye aye Sir!

**Comment by deluge** [comment]
Hy @stephrdev. Pycodestyle is implemented. I had to change the migration files to the expected pep8 line length. I was looking for a setting in setup.cfg but I can´t find any property. Do you have any idea?

**Comment by deluge** [comment]
Hy @stephrdev,
I removed pytest-flakes and pycodestyle and added pytest-flake8. Now, I think we are ready to merge.. Isn´t it?

**Pull request merged**